torch/ao/quantization/fx/_decomposed.py

```diff
@@ -32,6 +32,20 @@ def _quant_min_max_bounds_check(quant_min, quant_max, dtype):
 
 @impl(quantized_decomposed_lib, "quantize_per_tensor", "CompositeExplicitAutograd")
 def quantize_per_tensor(input, scale, zero_point, quant_min, quant_max, dtype):
+    """ Affine quantization for the Tensor using the same quantization parameters to map
+    from floating point to quantized values
+
+    Args:
+       input: original float32 Tensor
+       scale: quantization parameter for affine quantization
+       zero_point: quantization parameter for affine quantization
+       quant_min: minimum quantized value for output Tensor
+       quant_max: maximum quantized value for output Tensor
+       dtype: requested dtype for output Tensor
+
+    Returns:
+       per tensor quantized Tensor with requested dtype
```

Comment:
is this true?  I thought this returns a tensor of integers, which need to be combined with scale and zero_point to interpret the values.  "per tensor quantized tensor" sounds like "tensor + scale + zero_point", which doesn't match the function

Reply:
oh, makes sense, I'll explain this in a clearer way